.github/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+Please go the the `Preview` tab and select the appropriate PR template:
```

Comment:
Isnt' there a way to force a choice here when opening the PR? Maybe without this file and only files in `PULL_REQUEST_TEMPLATE`

Reply:
This is what we had before and somehow this is not working. It is also not clear for me how this should work...

Reply:
alright, it looks like the "select your template" thing only works for issues: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

This seems like it would work by opening the PR with the default template and then redirecting with a different template: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository and https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/using-query-parameters-to-create-a-pull-request for the `&template=...`. 

As written I think the first link will redirect to the same file, so maybe we can have three templates? One as the "please pick a template" and then the default one and the new architecture one. I'll check if there is a nicer way to do this

Reply:
unfortunately, that looks like the best solution for now: https://github.com/orgs/community/discussions/4620

Reply:
Which is what we have, right?

Reply:
not quite:

> As written I think the first link will redirect to the same file, so maybe we can have three templates? One as the "please pick a template" and then the default one and the new architecture one. I'll check if there is a nicer way to do this